src/unread/unreadHelpers.js

```diff
@@ -34,7 +34,7 @@ export const removeItemsDeeply = (objArray: Object[], messageIds: number[]): Obj
   );
 };
 
-const addItemsDeeply = (input: Object[], itemsToAdd: Object[], index: number): Object[] => {
+const addItemsDeeply = (input: Object[], itemsToAdd: any[], index: number): Object[] => {
```

Comment:
Thanks for the fix! Two remarks:

* Your commit message still says "we are limiting the types", which is completely baffling to the reader because it's giving a justification of why the type is specific, and in fact it's the opposite.

* There aren't actually any callers that pass objects -- this commit fixed all three callers to say they pass numbers. So this can just be `number`.